src/components/DiscordLogo.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from "react";
```

Comment:
This import is redundant since React17, right?

Reply:
Hey that's a good idea! Let me look into that.

Reply:
Got it! Check it out now @hauptrolle